core/server/data/validation/index.js

```diff
@@ -131,24 +128,17 @@ validateSettings = function validateSettings(defaultSettings, model) {
 };
 
 validateActiveTheme = function validateActiveTheme(themeName) {
```

Comment:
There are many, many more things that need to move into the theme package - I'm not sure what I'm going to do with validation yet. 

More PRs are coming, this one just brings us into line with LTS behaviour.